packages/now-next/test/fixtures/00-trailing-slash-add/now.json

```diff
@@ -0,0 +1,24 @@
+{
+  "version": 2,
+  "builds": [{ "src": "package.json", "use": "@vercel/next" }],
+  "probes": [
+    { "path": "/foo/", "status": 200, "mustContain": "foo page" },
+    {
+      "fetchOptions": { "redirect": "manual" },
+      "path": "/foo",
+      "status": 308,
+      "responseHeaders": {
+        "refresh": "/url=/foo/$/"
```

Comment:
Why check `refresh` instead of `location`?

The `$` seems unexpected 🤔

Reply:
We send both the location and refresh header, however, in tests, the `Location` header is absolute while the `Refresh` header is relative. Thus, it's easier to test the `Refresh` header.

Reply:
```
      cache-control=s-maxage=0
      connection=close
      content-type=text/plain
      date=Mon, 20 Jul 2020 20:03:01 GMT
      location=https://test2020-4ookam6q1.vercel.app/foo
      refresh=0;url=/foo
      server=Vercel
      strict-transport-security=max-age=63072000; includeSubDomains; preload
      transfer-encoding=chunked
      x-robots-tag=noindex
      x-vercel-id=iad1::8tkzw-1595275381290-9df066c957d2
```

Reply:
Strange, that must be a node-fetch feature then.

Reply:
An annoying feature at that!